mods/ts/maps/sunstroke/rules.yaml

```diff
@@ -0,0 +1,36 @@
+World:
```

Comment:
To fix the bridges you need to add an `ElevatedBridge` trait for each bridge on the map.
Use the map editor to find the top-left `Location`, the terrain `Height`, and the bridge `Length`.  The bridge starts and ends on the row of road cells that are part of the bridge head (use the cell grid to easily see these).  Set `Orientation` to `Y` for bridges running NE<->SW and `X` for bridges running NW<->SE.

E.g. for this bridge:

Set:
```yaml
		ElevatedBridgePlaceholder@a:
			Location: 60, -26
			Height: 5
			Orientation: Y
			Length: 8
```